src/plugins/expressions/common/expression_functions/series_calculation_helpers.ts

```diff
@@ -44,9 +44,13 @@ export function buildResultColumns(
   input: Datatable,
   outputColumnId: string,
   inputColumnId: string,
-  outputColumnName: string | undefined
+  outputColumnName: string | undefined,
+  options: { allowColumnOverride: boolean } = { allowColumnOverride: false }
```

Comment:
nit: Could we add the new option to the jsdoc so there's a brief explanation in the comments?